src/cli/cli_link_metrics.cpp

```diff
@@ -120,11 +129,12 @@ template <> otError LinkMetrics::Process<Cmd("query")>(Arg aArgs[])
     SuccessOrExit(error = otLinkMetricsQuery(GetInstancePtr(), &address, isSingle ? 0 : seriesId,
                                              isSingle ? &linkMetrics : nullptr, HandleLinkMetricsReport, this));
 
-    if (blocking)
```

Comment:
This specific command has a `block` as a parameter which seems to do the same? (looks like it is not documented though).
```blocking = (aArgs[3] == "block");```

Reply:
Sounds good to me. Thanks.